### PR TITLE
Transportation Event

### PR DESCRIPTION
**Pull request by vivienfan**
This PR adds yml for transportation event (a subevent extended from `Event`).

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransportEvent.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    title: Place
+    description: Information on where the event took place.
+    $ref: ./Place.yml
+    $linkedData:
+      term: place
+      '@id': https://schema.org/Place
+  actor:
+    title: Event Actors
+    description: >-
+      List of organizations acted on the event, https://w3id.org/traceability#Organization
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/TransportEvent.yml:35]:
```suggestion
      List of organizations which acted on the event, https://w3id.org/traceability#Organization
```

File: docs/openapi/components/schemas/common/TransportEvent.yml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+$linkedData:
+  term: TransportEvent
+  '@id': https://w3id.org/traceability#TransportEvent
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/TransportEvent.yml:3]:
```suggestion
  '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#Event
```

Comment (nissimsan) [reply docs/openapi/components/schemas/common/TransportEvent.yml:3]: The world has (more than) plenty of such shipping event schemes already.

**Review by nissimsan — CHANGES_REQUESTED** [review]

**Review by mprorock — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransportEvent.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ properties:
   actor:
     title: Event Actors
     description: >-
-      List of organizations acted on the event, https://w3id.org/traceability#Organization
+      List of organizations which acted on the event, https://w3id.org/traceability#Organization
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/TransportEvent.yml:35]:
```suggestion
      List of organizations which acted during the event, https://w3id.org/traceability#Organization
```

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransportEvent.yml (hunk 2 above)
Comment (mprorock) [reply docs/openapi/components/schemas/common/TransportEvent.yml:3]: also, concerned about Buy-Ship-Pay workflow mandates, etc

Comment (mprorock) [reply docs/openapi/components/schemas/common/TransportEvent.yml:3]: `https://service.unece.org/trade/uncefact/vocabulary/uncefact/#transportEvent`

**Comment by brownoxford** [comment]
Tabling for now, until @nissimsan is available for discussion.

**Comment by OR13** [comment]
I don't feel super strongly one way or the other, but changes to `@id` will break JSON-LD signatures... we might prefer to make these terms directly map to EPCIS if thats the intention. 

https://github.com/gs1/EPCIS/search?q=AggregationEvent

**Pull request merged**